### PR TITLE
Update README.md

### LINKED ISSUE #262: link to standard on main README page is broken.
This link no longer works and the README is semi broken.  Can we fix this?  https://uptane.org/uptane-standard/2.1.0/uptane-standard.html

I think this is related to the site switch over.

### PR DESCRIPTION
**Pull request by JustinCappos**
Fixes #262

### CODE REVIEW

**Review by mnm678 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Major and minor release dates are set by the Uptane Standards committee. All fin
 
 The Uptane Standards document should be considered the authoritative resource for the framework. Several other documents and materials are available or currently in development. The information in all of these other guidelines should be viewed as complementary to the official Uptane Standard, and as recommendations rather than mandatory instructions.
 
-* [Uptane Standard v.2.1.0](https://uptane.github.io/uptane-standard/2.1.0/uptane-standard.html
-* [Deployment Best Practices v.2.0.0](https://uptane.github.io/papers/V2.0.0_uptane_deploy.html)
+* [Uptane Standard v.2.1.0](https://uptane.org/docs/standard/uptane-standard)
+* [Deployment Best Practices v.2.0.0](https://uptane.org/docs/deployment/best-practices)
 * [Uptane POUF (Protocols, Operations, Usage, and Formats) Guidelines](https://uptane.github.io/pouf.html)
```
Comment (hexsecs) [inline README.md:17]:
```suggestion
* [Uptane POUF (Protocols, Operations, Usage, and Formats) Guidelines](https://uptane.org/docs/enhancements/pouf/pouf-main)
```

File: README.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ Major and minor release dates are set by the Uptane Standards committee. All fin
...
 * [Example POUF](https://uptane.github.io/reference_pouf.html)
```
Comment (hexsecs) [inline README.md:18]:
```suggestion
* [Example POUF](https://uptane.org/docs/enhancements/pouf/pouf1)
```

**Review by hexsecs — CHANGES_REQUESTED** [review]
There were a few other broken links. I added suggested fixes for them.

**Comment by hexsecs** [comment]
After some changes on the uptane.org site, there are still some broken links that need to be fixed.

**Comment by tkfu** [comment]
All outstanding fixes have been applied.

**Review by tkfu — APPROVED** [review]

**Review by hexsecs — APPROVED** [review]

**Pull request merged**